pkg/helm/run.go

```diff
@@ -49,14 +50,19 @@ func printVersion() {
 func Run(flags *hoflags.HelmOperatorFlags) error {
 	printVersion()
 
-	namespace, found := os.LookupEnv(k8sutil.WatchNamespaceEnvVar)
-	log = log.WithValues("Namespace", namespace)
+	watchNamespace, found := os.LookupEnv(k8sutil.WatchNamespaceEnvVar)
 	if found {
-		log.Info("Watching single namespace.")
+		log.Info("Watching single namespace", "watchNamespace", watchNamespace)
```

Comment:
Just a nit but the env could be set to `WATCH_NAMESPACE = ""` and this would log `Watching single namespace`.